week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,7 +25,16 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num == 'number' && num % 2 == 0 && num <=100 )
+  {
+    return true;
+  }
+  else {
+    return false;
+  };
```

Comment:
It's a correct solution but do we really need the if-else statement?

Reply:
we can write it like this I think:
`function validate(num) {
  return (typeof num == 'number' && num % 2 == 0 && num <=100 )
}`

Reply:
Exactly, as that expression results in a Boolean value. So you don't need to "re-specify" that it is true or false depending on the outcome, you can simply use the outcome of that expression.